src/app/settings/vault-grouping.test.ts

```diff
@@ -45,7 +46,7 @@ describe('vaultWeaponGroupingSelector', () => {
       {
         ...firstItem,
         typeName: undefined as unknown as string,
-        itemCategoryHashes: [0],
+        itemCategoryHashes: [0 as ItemCategoryHashes],
```

Comment:
Do we have an ItemCategoryHash.Invalid def we can use instead of 0 for any of the places we're using/comparing to 0?

Reply:
No - this test is intentionally using an invalid hash for some reason.

Reply:
Was also looking at https://github.com/DestinyItemManager/DIM/pull/10728/files#diff-f5fb26c29f2cdf8869105eb04a5d34ee01d285f2b4a163a23bf88488be6c0a86R108 re: making a definition for Invalid hash.

Reply:
Yeah, in that case it should be `undefined`, not zero. It's modeling the absence of a hash, not a special invalid hash.